backend/src/server/routes/v1/organization-router.ts

```diff
@@ -263,7 +264,18 @@ export const registerOrgRouter = async (server: FastifyZodProvider) => {
         enforceMfa: z.boolean().optional(),
         selectedMfaMethod: z.nativeEnum(MfaMethod).optional(),
         allowSecretSharingOutsideOrganization: z.boolean().optional(),
-        bypassOrgAuthEnabled: z.boolean().optional()
+        bypassOrgAuthEnabled: z.boolean().optional(),
+        userTokenExpiration: z
+          .string()
+          .regex(new RE2(/^\d+[mhdw]$/), "Must be a number followed by m, h, d, or w")
```

Comment:
this would actually still be problematic because zod would use the native regex engine. Can you instead just add custom logic to the refine method? You can manually use re2 there